packages/labs/react/src/create-component.ts

```diff
@@ -122,15 +122,19 @@ const setProperty = <E extends Element>(
   events?: EventNames
 ) => {
   const event = events?.[name];
-  if (event !== undefined) {
+  if (event !== undefined && value !== old) {
     // Dirty check event value.
-    if (value !== old) {
-      addOrUpdateEventListener(node, event, value as (e?: Event) => void);
-    }
-  } else {
-    // But don't dirty check properties; elements are assumed to do this.
-    node[name as keyof E] = value as E[keyof E];
+    addOrUpdateEventListener(node, event, value as (e?: Event) => void);
+    return;
   }
+
+  if (value === undefined && name in HTMLElement.prototype) {
```

Comment:
Wonder if we should do `value === undefined || value === null` also. Can we update the table to check the differences between null also?

Reply:
Updated the table to include the differences of `null`.
I didnt update the `value === undefined || value === null`. According to the table it doesn't change the behavior. I think this is mostly likely because a `null` value is passed via:
```
node[name as keyof E] = value as E[keyof E];
```

Reply:
It looks like the playground is not actually setting the `null` properties and is instead using the `undefined` ones. When that's corrected, it shows we need the `null` check as well: https://lit.dev/playground/#gist=b8209f138a222eceb45aa3aafc9301cb.

Reply:
will do! thank you